lib/jxl/dec_render_pipeline.h

```diff
@@ -24,28 +24,31 @@ enum class RenderPipelineChannelMode {
 
 class RenderPipelineStage {
  public:
-  // `input` points to `2*MaxPaddingY() + 1` pointers, each of which points to
-  // `3+num_non_color_channels` pointer-to-row. So, `input[MaxPaddingY()][0]` is
+  // `input` points to `2*MaxBorderY() + 1` pointers, each of which points to
```

Comment:
Mhhhh... I'd prefer to avoid negative offsets if possible. Also in most (all?) cases MaxPaddingY() should be a class constant, so I wouldn't be too concerned about getting it wrong.